indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -401,32 +411,40 @@
       (let [parsed-coll-concept (cp/parse-concept context coll-concept)]
         (index-concept context coll-concept parsed-coll-concept options)))))
 
-(defn- index-variable
-  "Index the associated variable concept of the given variable concept id."
-  [context variable-concept-id options]
-  (let [var-concept (meta-db/get-latest-concept context variable-concept-id)
-        parsed-var-concept (cp/parse-concept context var-concept)]
-    (index-concept context var-concept parsed-var-concept options)))
+(defn- concept-id->indexed-concept
```

Comment:
The arrow syntax usually refers to a data transformation. What about renaming `index-associated-concept`?

Reply:
Yeah, my thought was very loosely around this: "transforming" a concept id to a proper concept; your suggestion is a good one.

Reply:
Changed.